tests/sessions_tests/tests.py

```diff
@@ -713,3 +714,36 @@ def test_unpickling_exception(self):
 
         self.session.serializer = PickleSerializer
         self.session.load()
+
+
+class SessionSettingsTests(unittest.TestCase):
+
+    @override_system_checks([check_session_settings])
+    def test_language_cookie_name_and_session_cookie_name_defaults(self):
+        self.assertNotEqual(settings.SESSION_COOKIE_NAME, settings.LANGUAGE_COOKIE_NAME)
+        errors = checks.run_checks()
+        expected = []
+        self.assertEqual(errors, expected)
+
+    @override_settings(LANGUAGE_COOKIE_NAME='foo')
```

Comment:
You can put multiple settings in `@override_settings` -- don't need to call it twice.